app/controllers/api/base_controller.rb

```diff
@@ -23,9 +23,9 @@ class BaseController < ActionController::Base
 
     rescue_from NoMethodError do |error|
       Foreman::Logging.exception("Action failed", error)
-      message = _("Internal Server Error: the server was unable to finish the request. ")
-      message << _("This may be caused by unavailability of some required service, incorrect API call or a server-side bug. ")
-      message << _("There may be more information in the server's logs.")
+      message = _("Internal Server Error: the server was unable to finish the request. ") + \
+        _("This may be caused by unavailability of some required service, incorrect API call or a server-side bug. ") + \
```

Comment:
```suggestion
      message = _("Internal Server Error: the server was unable to finish the request. ") + 
        _("This may be caused by unavailability of some required service, incorrect API call or a server-side bug. ") + 
```
I don't think we need `\` here, since it just concatenates strings returned by a function call and simple `+` is enough. What I meant by `\` is to use it instead of multiple `+` or other func calls (such as `_()`). Something like: https://github.com/theforeman/foreman/blob/develop/app/controllers/api/v2/hosts_bulk_actions_controller.rb#L16-L18. But please, consider if it actually works well with the translation process. If the example is actually wrong, then we can simply go with `+` here.

Reply:
Perhaps that's just my issue with not using multi lines in Ruby that often. Is the backslash redundant there?

Reply:
Depending on how much changes do you want to do :D

1. We can use only one `_()` call with the strings being concatenated by `\`. More changes and we need to verify if it doesn't break the translation, but we already do it in the example above.
2. We can leave current `_()` calls and concatenate results with `+`. Less changes, essentially changing `message` variable's string modification via `<<` to changing `message` variable to be a single string preconcatenated by `+`. 

Using both `\` and `+` works for some reason, but I wouldn't mix them...

Reply:
> We can use only one `_()` call with the strings being concatenated by `\`. More changes and we need to verify if it doesn't break the translation, but we already do it in the example above.

I was hesitant to change the translation but it may end up being easier for translators if they get the whole context.